package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@sugarcrm/ventana",
-  "version": "1.0.3",
+  "version": "1.0.4",
```

Comment:
Sorry, can you undo the change to the version number?

We'll update the version number right before publishing a new release in a separate commit.